tests/unit/ops/aio/test_gds.py

```diff
@@ -54,7 +54,7 @@ def _get_test_write_file_and_device_buffer(tmpdir, ref_buffer, gds_handle, index
 def _validate_handle_state(handle, single_submit, overlap_events):
     assert handle.get_single_submit() == single_submit
     assert handle.get_overlap_events() == overlap_events
-    assert handle.get_thread_count() == IO_PARALLEL
+    assert handle.get_thread_count() == 1
```

Comment:
Why is `IO_PARALLEL` no longer correct?

Reply:
For GDS the number of threads for `IO_PARALLEL` are set in the `cufile.json`. From the handle's point of view there is still only 1 thread. This [PR](https://github.com/microsoft/DeepSpeed/pull/6459/files) has the corresponding change in the GDS handle.